src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Financial/RoatpFinancialApplicationViewModel.cs

```diff
@@ -10,7 +10,7 @@ namespace SFA.DAS.AdminService.Web.ViewModels.Apply.Financial
 {
     public class RoatpFinancialApplicationViewModel : OrganisationDetailsViewModel
     {
-        public List<Section> Sections { get; }
+        public List<Section> Sections { get; set; }
```

Comment:
I couldn't see why the set is required here. Is there a need to replace/initialise the entire list?

Reply:
yes, I need to map them back in the ConvertFinancialApplicationToFinancialClarificationViewModel

Reply:
which inherts from RoatpFinancialApplicationViewModel